aliyun/src/main/java/org/apache/iceberg/aliyun/oss/BaseOSSFile.java

```diff
@@ -23,16 +23,19 @@
 import com.aliyun.oss.OSSErrorCode;
 import com.aliyun.oss.OSSException;
 import com.aliyun.oss.model.SimplifiedObjectMeta;
+import org.apache.iceberg.aliyun.AliyunProperties;
 import org.apache.iceberg.relocated.com.google.common.base.MoreObjects;
 
 abstract class BaseOSSFile {
   private final OSS client;
   private final OSSURI uri;
+  private final AliyunProperties aliyunProperties;
   private SimplifiedObjectMeta metadata;
 
-  BaseOSSFile(OSS client, OSSURI uri) {
+  BaseOSSFile(OSS client, OSSURI uri, AliyunProperties aliyunProperties) {
```

Comment:
Could we keep the original constructor and introduce an extra `BaseOSSFile` constructor with the newly introduced  `aliyunProperties` argument so that we don't need to change the unrelated classes (such as `TestOSSInputFile`, `OSSInputFile` etc) ?